### PR TITLE
Fix err500 clarity

### LINKED ISSUE #95: 500 error in Details action after acceptance of clarification by RM
Sorry for reporting another urgent bug.
After the Register Manager accepts a submitted clarification action for an item, the Details action for that item gives a HTTP Status 500 error in either the Submitting Organization, Register Manager or Control Body folders. No one is able to view the item's details once the RM accepts and the CB cannot evaluate the proposal. The Details action works before the RM accepts it.

### PR DESCRIPTION
**Pull request by phuonghuynh**
Fix #95

record link https://recordit.co/fuBTs0d9Ol

### CODE REVIEW

**Comment by ronaldtse** [comment]
Thank you @phuonghuynh !

File: src/iso-registry-client/src/main/webapp/WEB-INF/views/registry/registers/gcp/infosrc_panel.html [hunk 1]
```diff
@@ -87,7 +87,7 @@
 					<div class="modal-header">
 						<h4 th:text="#{popup.informationSource.header}"></h4>
 					</div>
-					<div th:if="!${isProposal} or ${isReadOnly}" class="modal-body" th:object="${item.informationSource[__${rowstat.index}__]}">
+					<div th:if="!${isProposal} or ${isReadOnly}" class="modal-body" th:object="${proposal.informationSource[__${rowstat.index}__]}">
```
Comment (florianesser) [inline src/iso-registry-client/src/main/webapp/WEB-INF/views/registry/registers/gcp/infosrc_panel.html:90]:
```suggestion
					<div th:if="!${isProposal}" class="modal-body" th:object="${item.informationSource[__${rowstat.index}__]}">
```

File: src/iso-registry-client/src/main/webapp/WEB-INF/views/registry/registers/gcp/infosrc_panel.html [hunk 2]
```diff
@@ -87,7 +87,7 @@
...
 						<div th:include="registry/registers/gcp/infosrc_panel_content :: informationSourcePanelContent(${rowstat.index})"/>
 					</div>
 					<div th:if="${isProposal} and !${isReadOnly}" class="modal-body" th:object="${proposal.informationSource[__${rowstat.index}__]}" th:with="isReadOnly='true'">
```
Comment (florianesser) [inline src/iso-registry-client/src/main/webapp/WEB-INF/views/registry/registers/gcp/infosrc_panel.html:93]:
```suggestion
					<div th:if="${isProposal}" class="modal-body" th:object="${proposal.informationSource[__${rowstat.index}__]}" th:with="isReadOnly='true'">
```

**Comment by phuonghuynh** [comment]
Thanks @florianesser , appended your suggestion to this PR.

File: src/iso-registry-client/src/main/webapp/WEB-INF/views/registry/registers/gcp/infosrc_panel.html (hunk 2 above)
Comment (florianesser) [reply src/iso-registry-client/src/main/webapp/WEB-INF/views/registry/registers/gcp/infosrc_panel.html:93]: @phuonghuynh My old code here really makes not much sense. I think this is also what's causing that editing  citations is not possible in a clarification. For this PR it's not necessary to change it, though.

**Pull request merged**